e2e/test_operator_upgrades/operator_upgrades_test.go

```diff
@@ -315,6 +315,7 @@ var _ = Describe("Operator Upgrades", Label("e2e", "pr"), func() {
 		fixtures.GenerateUpgradeTableEntries(testOptions),
 	)
 
+	// TODO double check failure!!
```

Comment:
is this for this PR or post-merge reviewing?

Reply:
That's an old comment I forgot to remove 🙈